ros2topic/ros2topic/verb/pub.py

```diff
@@ -81,9 +99,16 @@ def publisher(
         node_name = NODE_NAME_PREFIX + '_publisher_%s' % (message_type.replace('/', '_'), )
     rclpy.init()
 
+    # Build a QoS profile based on user-supplied arguments
+    profile = rclpy.qos.qos_profile_system_default
+    if qos_durability:
+        profile.durability = QOS_DURABILITY_OPT[qos_durability]
+    if qos_reliability:
+        profile.reliability = QOS_RELIABILITY_OPT[qos_reliability]
+
     node = rclpy.create_node(node_name)
 
-    pub = node.create_publisher(msg_module, topic_name, 10)
+    pub = node.create_publisher(msg_module, topic_name, qos_profile=profile)
```

Comment:
This is deprecated. Prefer:
```suggestion
    pub = node.create_publisher(msg_module, topic_name, profile)
```